### ISSUE TITLE
JWT verification, documentation different from the implementation #KJUR.jws.JWS.verifyJWT

### ISSUE THREAD

**Issue by esynaps** [kjur/jsrsasign#318]
Hi, I was faced with a small problem when checking a JWT. 

The implementation does not match the documentation for acceptFields. 

The aud parameter expects either an array or a string from the documentation, whereas in reality if it is a string that is passed as a parameter, there is no verification of the claim. Even if the string does not match the claim of the token, the latter is still considered correct when it is not.

Documentation:
https://kjur.github.io/jsrsasign/api/symbols/KJUR.jws.JWS.html#.verifyJWT

> Here is available acceptField argument parameters:
> alg - array of acceptable signature algorithm names (ex. ["HS256", "HS384"])
> iss - array of acceptable issuer names (ex. ['http://foo.com'])
> sub - array of acceptable subject names (ex. ['mailto:john@foo.com'])
> aud - **array or string** of acceptable audience name(s) (ex. ['http://foo.com'])
> 

Src: 
https://github.com/kjur/jsrsasign/blob/master/src/jws-3.3.js
lines 693 - 702

```
    // 7. audience ('aud' in payload) check
    if (pPayload.aud !== undefined && typeof acceptField.aud === "object") {
	if (typeof pPayload.aud == "string") {
	    if (! _inArray(pPayload.aud, acceptField.aud))
		return false;
	} else if (typeof pPayload.aud == "object") {
	    if (! _includedArray(pPayload.aud, acceptField.aud))
		return false;
	}
    }
```

**Comment by kjur** [comment]
Thank you for your report. I'll fix the document in the next version.

**Comment by kjur** [comment]
I've released 8.0.5 today.
